pkg/envoy/proxy.go

```diff
@@ -59,10 +60,14 @@ type ProxyConfig struct {
 func NewProxy(cfg ProxyConfig) Proxy {
```

Comment:
Would also be great to rename ProxyConfig to EnvoyConfig here - and pass the 'real' ProxyConfig as param. 

But first we need to bike-shed if we'll merge with Sidecar and/or rename.